Description: disable the overwrite feature, as it overwrites source code and rust sources are installed without write permissions for the user.

Index: io-uring/Cargo.toml
===================================================================
diff --git a/Cargo.toml b/Cargo.toml
index 6cca674..1250e07 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -53,3 +53,3 @@
 io_safety = []
-overwrite = ["bindgen"]
+#overwrite = ["bindgen"]
 
